Services/data/devicemodels/scripts/StartElevator-method.js

```diff
@@ -19,8 +18,14 @@
 /*jslint unparam: true*/
 function main(context, previousState, previousProperties) {
 
-    log("Executing JavaScript StartElevator function.");
+    log("Executing JavaScript StartElevator method.");
+
+    var state = {
+        moving: true
```

Comment:
Who actually changes props that show the device is moving? i.e. does the simulation read this property and change the floor if it's set?  I'm surpirsed the method doesn't take a param with desired floor for the elevator.  Does this meet the requirements laid out?